spec/features/users/login_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+RSpec.describe 'As a visitor' do
```

Comment:
'As a visitor' doesn't quite make sense in the context of this test, maybe we could say something along the lines of 'when I hit (specific path)'